compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmPackageGen.java

```diff
@@ -301,6 +301,18 @@ private static void setModuleStatusField(ClassWriter cw, MethodVisitor mv, Strin
 
         mv.visitInsn(ICONST_0);
         mv.visitFieldInsn(PUTSTATIC, initClass, MODULE_STARTED, "Z");
+
+        fv = cw.visitField(ACC_PUBLIC + ACC_STATIC, MODULE_START_PARENT_ATTEMPTED, "Z", null, null);
+        fv.visitEnd();
+
```

Comment:
Shall we remove unnecessary new lines

Reply:
Fixed in 8da28c04c1397a39613ef773b0f56c5989ce3016.